src/Worker.Extensions.DurableTask/Worker.Extensions.DurableTask.csproj

```
@@ -42,6 +42,7 @@
     <PackageReference Include="Microsoft.DurableTask.Client.Grpc" />
     <PackageReference Include="Microsoft.DurableTask.Worker.Grpc" />
     <PackageReference Include="Grpc.Net.Client" />
+    <PackageReference Include="Microsoft.DurableTask.Analyzers" />
```

I usually see `PrivateAssets="all"` when analyzer packages are referenced like this. Do we need something similar?
```suggestion
    <PackageReference Include="Microsoft.DurableTask.Analyzers" PrivateAssets="All" />
```

---

I added `PrivateAssets="All"` to the Analyzer package reference and tested it in a .NET Isolated DF app, but the analyzer isn't working when I add that. It's also not showing up in the Analyzer section in Solution Explorer. 

It works as expected when I test without `PrivateAssets="All"` though. It looks like adding `PrivateAssets` doesn't pass the analyzer dependency to projects that reference the Worker extension package.

I also checked [how we reference the .NET In Proc Analyzer in WebJobs.Extensions.DurableTask.csproj](https://github.com/Azure/azure-functions-durable-extension/blob/9b39310c869c8e02e3d3b7963e70803c9d53cc9d/src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj#L61) and it doesn't have `PrivateAssets="All"` either so I think we don't need it.

---

OK, thanks for checking!